ui/run.sh

```diff
@@ -149,6 +155,8 @@ case "$(basename ${0})" in
         o "Pinging Management API Endpoint until ready"
         run ecsconfig ping -c -x || exit $?
         run ecsconfig sp -a || exit $?
+	o "Sleeping for 15 minutes for Storage Pool to be ready"
```

Comment:
No, use the correct ping tool or write an additional ping function if `ecsconfig ping` is not working for you in its current state.